### PR TITLE
Add dhcp_browser plugin

### PR DESCRIPTION
**Pull request by brandonweeks**

### CODE REVIEW

**Comment by mmoll** [comment]
Please add a spec test :)

**Comment by brandonweeks** [comment]
Added in 5d03a6f.

File: spec/classes/foreman_plugin_dhcp_browser_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::hooks' do
```
Comment (mmoll) [inline spec/classes/foreman_plugin_dhcp_browser_spec.rb]: hooks -> dhcp_browser

**Comment by mmoll** [comment]
please fix the copy/paste error and squash everything together into one commit, looking good otherwise.

**Comment by brandonweeks** [comment]
That's embarrassing. 😳

Fixed.

**Comment by mmoll** [comment]
please squash also the two remaining commits together, :+1: pending tests then :)

**Comment by brandonweeks** [comment]
And squashed.

**Comment by ekohl** [comment]
:+1:

**Comment by domcleal** [comment]
Merged, thanks!

**Pull request merged**